{{cookiecutter.project_name}}/monitoring/README.md

```diff
@@ -0,0 +1,72 @@
+# Setup Data Monitoring for Inference Table
+
+**Databricks data monitoring is currently in private review. Please `sign up` for the private review and get approved 
+before proceeding.**
+
+This directory contains the notebook that can run in databricks workspace
+to set up data monitoring for prod inference table.
+
+## Prerequisites
+
+### Unity Catalog
+The table to be monitored must exist in Unity Catalog.
+If Unity Catalog is enabled for your workspace but the table exists in metastore, please copy the table to Unity Catalog.
+
+For example, the following code shows how to import your data to a Unity Catalog managed table
+ - load data from metastore example_project_predictions
+ - write data to Unity Catalog my_unity_catalog.my_schema.my_inference_table
+ - convert prediction column to double type
+ - convert tpep_pickup_datetime to timestamp type
+ - convert tpep_dropoff_datetime to timestamp type
```

Comment:
Are these instructions relevant for showing how to import data into UC? Seems orthogonal

Reply:
It describes the example, I wanted to indicate that changes can also be applied when the table is copied to UC. 
Let me remove them.